.github/workflows/update_knapsack_report.yml

```diff
@@ -0,0 +1,69 @@
+name: Update Knapsack Report
+
+# This workflow should be scheduled at certain intervals
+on:
+ schedule:
+     - cron: '0 5 31 2 *'    
+# The above cron does not run. Replace with the wanted periodicity.
```

Comment:
Would be possible to make the `ci` process schedule this job dynamically? E.g. Knapsack reports will be updated 1 hour after the ci for master branch runs